tests/test_fellowships.py

```diff
@@ -1,37 +1,111 @@
 import pytest
 import numpy as np
+from torch.utils.data import DataLoader
 
 from continuum import ClassIncremental, InstanceIncremental
-from continuum.datasets import InMemoryDataset
 from continuum.datasets import (
-    CIFAR10, CIFAR100, KMNIST, MNIST, CIFARFellowship, FashionMNIST, Fellowship, MNISTFellowship
+    CIFAR10, CIFAR100, KMNIST, MNIST, CIFARFellowship, FashionMNIST, Fellowship, MNISTFellowship,
+    InMemoryDataset, Fellowship
 )
 
 
+@pytest.fixture
+def dataset7c():
+    return InMemoryDataset(*gen_dataset(7, 0))
+
+
+@pytest.fixture
+def dataset10c():
+    return InMemoryDataset(*gen_dataset(10, 1))
+
+
+@pytest.fixture
+def dataset20c():
+    return InMemoryDataset(*gen_dataset(20, 2))
+
+
+def gen_dataset(nb_classes, pixel_value):
+    nb_items_per_class = 5
+
+    x_train = np.ones((nb_items_per_class * nb_classes, 32, 32, 3)) * pixel_value
+    y_train = []
+    for i in range(nb_classes):
+        y_train.append(np.ones(nb_items_per_class) * i)
+    y_train = np.concatenate(y_train)
+
+    return (x_train, y_train)
+
+
+@pytest.mark.parametrize("update_labels,increment", [
+    (True, 1),
+    (True, [7, 10, 20]),
+    (False, 1),
+    (False, [7, 10, 20]),
+])
+def test_inMemory_Fellowship(update_labels, increment, dataset7c, dataset10c, dataset20c):
+    fellow = Fellowship([dataset7c, dataset10c, dataset20c], update_labels=update_labels)
+
+    x, y, t = fellow.get_data()
+    assert len(np.unique(t)) == 3
+    if update_labels:
+        assert len(np.unique(y)) == 7 + 10 + 20
```

Comment:
assert len(np.unique(y)) == np.array(increment).sum()
ce serait moins hard coded comme ca (pareille pour la suite)

Reply:
Actually, not we cannot do it, if the increment == 1 it doesn't work. Hardcoded is not that bad imo